### DISCUSSION TITLE
Flexget WebUI loads then turns to blank page

### DISCUSSION THREAD

**Discussion by celloch** [Flexget/Flexget#3819] [Q&A]
I have been using Flexget for years without a single issue. 
Today, out of nowhere, it started displaying a blank page after logging in. You can briefly see the page loading its elements.

Inspecting the page, I get an error saying
Invalid version: 3.9

Flexget works, it downloads stuff in the background. It's just the Web interface which is not working.

https://github.com/Flexget/Flexget/assets/6252801/4d641db4-15bc-4fcd-b3f3-8207a2ee99d4

**Comment by gazpachoking** [comment]
Sounds like probably to do with this. https://github.com/Flexget/webui/issues/182 A new release should be out in a few hours which should fix the issue.

**Comment by celloch** [comment]
Still not working for me on 3.9.2 after upgrading.
Tried uninstalling, deleting pip cache, installing again. Same behaviour. Funny enough, even on 3.9.2 the error says Invalid version: 3.9
Everything works on a brand new installation on a different computer (both Mac computers).

**Reply by gazpachoking** [reply]
Did you make sure to stop and start your daemon after upgrade?

**Reply by celloch** [reply]
I did. After testing more, I just noticed that when the computer restarts (and Flexget is loaded automatically using flexgetdaemon application as a Login item) the problem is still there.
But if I stop the daemon from CLI and restart ("flexget daemon start) then the Web UI works as expected.

**Comment by gazpachoking** [comment]
I'm guessing maybe you have 2 installs, one that's getting launched at
startup, and one installed some other way that's the new version

**Reply by celloch** [reply]
Yes that was probably the case, after uninstalling I still had flexget in /usr/local/bin, so I did a full cleanup of every instance.

I think I have a mess in my system though. After installing again (I got rid of pip and installed it again too), flexget is found in: 

> /System/Volumes/Data/opt/homebrew/bin/flexget
> /System/Volumes/Data/opt/homebrew/lib/python3.11/site-packages/flexget
> /opt/homebrew/bin/flexget
> /opt/homebrew/lib/python3.11/site-packages/flexget

instead of the paths mentioned in the Autorun documentation.

So I'm not able to use the plist method (it doesn't work if I insert the homebrew path in the plist file, the process is running but the web server is unavailable) or the launcher application.
Anyway no big deal, I will run it manually as a daemon in the terminal window. The server is almost always up anyway and restarts are rare.
